src/transformStateWithClones.js

```diff
@@ -7,7 +7,39 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const result = [];
+  let currentState = { ...state };
+
+  for (const action of actions) {
+    let temporalState = { ...currentState };
```

Comment:
maybe better use `currentState` variable for this? i think this is redundant variable

Reply:
thank, I fixed it all